apps/client/src/styles.css

```diff
@@ -1 +1,13 @@
-/* You can add global styles to this file, and also import other style files *//*# sourceMappingURL=styles.css.map */
\ No newline at end of file
+@tailwind components;
+@tailwind base;
+@tailwind utilities;
+
+@import url('https://fonts.googleapis.com/css2?family=Major+Mono+Display&family=Roboto+Mono:wght@400;500&family=Sora:wght@400;600&display=swap');
+
+/* font-family: 'Major Mono Display', monospace;
+font-family: 'Roboto Mono', monospace;
+font-family: 'Sora', sans-serif;   */
+
```

Comment:
shall we remove these commented code

Reply:
This was removed from PR #43 under commit 31bd36e8f931f9879a23fc1eeca7223db3390113